src/main/java/org/elasticsearch/search/highlight/HighlightPhase.java

```diff
@@ -91,6 +93,7 @@ public void hitExecute(SearchContext context, HitContext hitContext) {
                 }
             }
 
+            boolean fieldNameContainsWildcards = field.field().contains("*");
```

Comment:
There should be a "resolve" button in github like there is in googledocs so we can ignore long resolved recommendations.